build.gradle

```diff
@@ -60,7 +60,7 @@ repositories {
 
 dependencies {
     implementation 'log4j:log4j:1.2.17',
-            'com.emc.ecs:smart-client:2.2.1',
+            'com.emc.ecs:smart-client:2.3.0',
             'com.emc.ecs:object-transform:1.1.0',
```

Comment:
since we are using the Jackson annotations directly, we need to add the `com.fasterxml.jackson.module:jackson-module-jaxb-annotations:2.12.2` library as an `implementation` dependency

Reply:
done